ras/hwinfo.py

```diff
@@ -71,8 +72,9 @@ def test(self):
         self.run_cmd("hwinfo --debug 0 --disk --log=-")
         self.run_cmd("hwinfo --short --block")
         self.run_cmd("hwinfo --disk --save-config=all")
-        if "failed" in process.system_output("hwinfo --disk --save-config=all | "
-                                             "grep failed | tail -1", shell=True):
+        if "failed" in process.system_output("hwinfo --disk --save-config"
+                                             "=all | ""grep failed | tail -1",
```

Comment:
can the above line written as `"=all | grep failed | tail -1",` instead ?

Reply:
sure , any way i am planning to use native python to parse output